coverage/clover.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<coverage generated="1498995873564" clover="3.2.0">
```

Comment:
coverage dir should be ignored, it will automatically build after run test.
too much useless files